script.js

```diff
@@ -50,4 +50,393 @@ menuIcon.addEventListener('keydown', (e) => {
+}
+
+for (let i = 1; i < 2; i += 1) {
+  const popupBackground = document.querySelector('.popup_background');
+  const popupWindow = document.querySelector('.popup');
+
+  const closeBtn = document.createElement('div');
+  closeBtn.classList.add('close_buttons');
+
+  const popupCloseMobile = document.createElement('img');
+  popupCloseMobile.src = projectInfo[i].closeBtnMobile;
+  popupCloseMobile.classList.add('close_mobile');
+
+  const popupCloseDesktop = document.createElement('img');
+  popupCloseDesktop.src = projectInfo[i].closeBtnDesktop;
+  popupCloseDesktop.classList.add('close_desktop');
+
+  closeBtn.append(
+    popupCloseMobile,
+    popupCloseDesktop,
+  );
+
+  const popupImg = document.createElement('div');
+  popupImg.classList.add('popup_images');
+
+  const popupMobileImage = document.createElement('img');
+  popupMobileImage.src = projectInfo[i].popupImageMobile;
+  popupMobileImage.classList.add('mobile_popup_image');
+
+  const popupDesktopImage = document.createElement('img');
+  popupDesktopImage.src = projectInfo[i].popupImageDesktop;
+  popupDesktopImage.classList.add('desktop_popup_image');
+
+  popupImg.append(
+    popupMobileImage,
+    popupDesktopImage,
+  );
+
+  const containerForHeaderContainerAndPopupButtons = document.createElement('div');
+  containerForHeaderContainerAndPopupButtons.classList.add('container_for_header_container_and_popup_buttons');
+
+  const headerContainer = document.createElement('div');
+  headerContainer.classList.add('header_container');
+
+  const popupHeader = document.createElement('h3');
+  popupHeader.classList.add('popup_header');
+  popupHeader.innerHTML = projectInfo[i].popupCardHeader;
+
+  headerContainer.appendChild(
+    popupHeader,
+  );
+
+  const popupButtonsDesktop = document.createElement('div');
+  popupButtonsDesktop.classList.add('popup_buttons_desktop');
+
+  const seeSourceBtn = document.createElement('button');
+  seeSourceBtn.type = 'button';
+  seeSourceBtn.classList.add('see-project');
+  seeSourceBtn.innerHTML = `
+        See Source
+          <span><img src="assets/popup/icons/see-source-icon.svg"></span>
+    `;
+
+  const seeLiveBtn = document.createElement('button');
+  seeLiveBtn.type = 'button';
+  seeLiveBtn.classList.add('see-project');
+  seeLiveBtn.innerHTML = `
+        See Live
+          <span><img src="assets/popup/icons/see-live-icon.svg"></span>
+    `;
+
+  const seeLiveBtnDesktop = document.createElement('button');
+  seeLiveBtnDesktop.type = 'button';
+  seeLiveBtnDesktop.classList.add('see-project');
+  seeLiveBtnDesktop.innerHTML = `
+        See Live
+          <span><img src="assets/popup/icons/see-live-icon.svg"></span>
+    `;
+
+  const seeSourceBtnDesktop = document.createElement('button');
+  seeSourceBtnDesktop.type = 'button';
+  seeSourceBtnDesktop.classList.add('see-project');
+  seeSourceBtnDesktop.innerHTML = `
+        See Source
+          <span><img src="assets/popup/icons/see-source-icon.svg"></span>
+    `;
```

Comment:
- [ ] Great job making your popup buttons. Kindly also include the button links such that when a user clicks on the button for seeing live, it takes him / her to the respective projects live or GitHub repository.
- [ ] Also, kindly reduce the padding in the buttons and increase the padding from the margins to the buttons in mobile view. Right now the design does not match the provided Figma design.

**This is how your Mobile version looks like**

![image](https://user-images.githubusercontent.com/86318284/202445688-69dc419c-4b4a-4520-b089-6ffaad409664.png)


**This is the desired design:**

![image](https://user-images.githubusercontent.com/86318284/202445814-c999cd5e-2180-4faa-af8c-bf42decc5796.png)